js/ui/appDisplay.js

```diff
@@ -1665,7 +1665,9 @@ const AppIcon = new Lang.Class({
     },
 
     getDragActor: function() {
-        return this.app.create_icon_texture(Main.overview.dashIconSize);
+        // This is a temporary change not to depend on the dash while
+        // we don't implement our own Drag'n'Drop mechanism in EOS.
+        return this.app.create_icon_texture(64);
```

Comment:
WHAT? I did this change yesterday and I'm pretty sure I pushed it... but you're right, it's not there, and I'm very confused...